src/main/java/mate/controller/AddCarController.java

```diff
@@ -0,0 +1,46 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.DriverService;
+import mate.service.ManufacturerService;
+
+public class AddCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final ManufacturerService manufacturerService
+            = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String carModel = req.getParameter("car_model");
+        Long idManufacturer = Long.valueOf(req.getParameter("manufacturer_id"));
```

Comment:
```suggestion
        Long manufacturerId = Long.valueOf(req.getParameter("manufacturer_id"));
```
This is the proper way to name model properties. Just as you did it above)